### DISCUSSION TITLE
ae扫光效果是否支持占位图替换

### DISCUSSION THREAD

**Discussion by lou-123456** [Tencent/libpag#2518] [Q&A] [answered]
扫光效果显示要用bmp，但是用了bmp又不能替换占位图，有解决方法吗

**Comment by kevingpqi123** [comment, marked as answer]
这是 pag 的 AE 特性支持列表：https://pag.io/zh-CN/feature.html#ae
 如果扫光效果作用的是图片图层，这里需要转成 BMP 不能替换占位图，如果扫光效果作用的不是图片图层，是支持替换替换的

**Reply by lou-123456** [reply]
![Clipboard - 2024-10-09 18 09 13](https://github.com/user-attachments/assets/212b4822-aba0-4899-869f-88b9db2beed4)
![Clipboard - 2024-10-09 18 10 45](https://github.com/user-attachments/assets/fa635174-c6fa-4df9-9ee6-18774553fad0)
我在矢量图层上用扫光依旧显示要转成bmp，这是为什么？

**Reply by kevingpqi123** [reply]
你看下这个文档：https://pag.io/zh-CN/feature.html#ae， 这种 AE 特性目前 pag 没有支持，只能通过 BMP 预合成， 我们支持了常用的一些 AE 特效以及 特性。
